Cargo.toml

```diff
@@ -88,6 +88,9 @@ alloy = { version = "0.1", features = [
     "node-bindings",
     "rpc-types-debug",
     "rpc-types-trace",
+    "rpc-types",
+    "json-rpc",
+    "rpc-client",
```

Comment:
`rpc-types` should not be necessary as it is enabled by any `rpc-types-*` flag automatically